### PR TITLE
run_tracker merged in taca

### PR DESCRIPTION
**Pull request by guillermo-carrasco**
This implements `taca analysis demultiplex`.

Can be improved with more options in the future, but this works.

### CODE REVIEW

File: taca/analysis/analysis.py [hunk 1]
```diff
@@ -206,26 +193,26 @@ def prepare_sample_sheet(run, config):
                  "SampleSheet.csv was already present in {} !!".format(FCID, FCID_samplesheet_dest)))
         return False
 
-    FCID_samplesheet_origin_dict = sampleSheetToDict(FCID_samplesheet_origin) # store samplesheet in a dict
+    FCID_samplesheet_origin_dict = samplesheet_to_dict(FCID_samplesheet_origin) # store samplesheet in a dict
```
Comment (vezzi) [inline taca/analysis/analysis.py]: @guillermo-carrasco WhyYouChangedMyWonderfullC++WayToNameFunctions!!!!!

**Comment by vezzi** [comment]
Considering the way in which you have renamed the functions I am tempted to not merge this pull request

File: taca/analysis/analysis.py (hunk 1 above)
Comment (guillermo-carrasco) [inline taca/analysis/analysis.py]: sorry dude, its not you... its me... you know...

**Comment by guillermo-carrasco** [comment]
haha thanks :-D

**Pull request merged**